src/views/BackgroundInfoView.vue

```diff
@@ -0,0 +1,417 @@
+          <!-- @todo: copy -->
+          Thank you for submitting your background information. Our students,
+          and school partners, are interested in learning more about the
+          volunteers at UPchieve!
+        </p>
+      </div>
+      <form class="background-info__form" @submit="submitForm" v-else>
+        <p class="background-info__subheader">
+          Our students, and school partners, are interested in learning more
+          about the volunteers at UPchieve! Please fill in the following
+          background information- who knows, it might help us pair you with the
+          right student!
+        </p>
+
+        <ol>
+          <li class="uc-form-col">
+            <p>
+              I am currently...<span class="background-info__question-required"
+                >*</span
+              >
+            </p>
+            <p class="background-info__question-description">
+              Select all that apply.
+            </p>
+            <p v-if="showInputErrors && occupation.length === 0" class="error">
+              Please fill out this field.
+            </p>
+            <div
+              class="uc-form-checkbox"
+              v-for="option in options.occupations"
+              :key="option"
+            >
+              <input
+                type="checkbox"
+                :value="option"
+                v-model="occupation"
+                :id="option"
+              />
+              <label class="uc-form-label" :for="option">
+                {{ option }}
+              </label>
+            </div>
+          </li>
+
+          <li class="uc-form-col">
+            <p>
+              Do you identify with any of the following statements?<span
+                class="background-info__question-required"
+                >*</span
+              >
+            </p>
+            <p class="background-info__question-description">
+              Select all that apply.
+            </p>
+            <p v-if="showInputErrors && background.length === 0" class="error">
+              Please fill out this field.
+            </p>
+
+            <div
+              class="uc-form-checkbox"
+              v-for="option in options.background"
+              :key="option"
+            >
+              <input
+                type="checkbox"
+                :value="option"
+                v-model="background"
+                :id="option"
+              />
+              <label class="uc-form-label" :for="option">
+                {{ option }}
+              </label>
+            </div>
+          </li>
+
+          <li class="uc-form-col">
+            <label for="linkedin" class="uc-form-label">
+              If you have one, please provide us with a link to your LinkedIn
+              profile.
+            </label>
+            <p class="background-info__question-description">(optional)</p>
+
+            <input
+              type="text"
+              pattern=".*linkedin\.com.*\/in\/.*"
```

Comment:
With the pattern attribute, you can use the `:invalid` pseudo selector for changing the input's border on non-matches
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern#Examples

Reply:
This is awesome! thank you